pkg/models/base_struct_user.go

```diff
@@ -18,6 +20,7 @@ func NewBaseStructUser(createdBy uuid.UUID) BaseStructUser {
 	return BaseStructUser{
 		createdByRelation: createdByRelation{
 			CreatedBy: createdBy,
+			CreatedAt: time.Now().UTC(),
```

Comment:
maybe we could add this to the DB as well? `NOT NULL DEFAULT NOW()`? - not for this PR though, but maybe in the future just for safety

Reply:
Yeah there's also `CURRENT_TIMESTAMP` I thought about adding in the SQL query we could do, but wasn't sure if we should obfuscate that in the DB vs showing explicitly in the code. We could get together and talk about the best convention if you want.

Reply:
i like your thinking - better in two places than implicit only in the DB